xml/deployment_scenarios.xml

```diff
@@ -0,0 +1,888 @@
+      </para>
+      <itemizedlist>
+       <listitem>
+        <para>
+         <literal>2</literal> Host machines for the mirror servers.
+        </para>
+        <itemizedlist>
+         <listitem>
+          <para>
+           SLES 15
+          </para>
+         </listitem>
+         <listitem>
+          <para>
+           2 (v)CPU
+          </para>
+         </listitem>
+         <listitem>
+          <para>
+           8 GB RAM
+          </para>
+         </listitem>
+         <listitem>
+          <para>
+           500 GB Storage
+          </para>
+         </listitem>
+        </itemizedlist>
+       </listitem>
+      </itemizedlist>
+     </listitem>
+    </varlistentry>
+    <varlistentry>
+     <term>Internal</term>
+     <listitem>
+      <para>
+       These machines will host the <literal>&rmtool;</literal> for RPM packages,
+       and <literal>Docker Container Registry</literal> for container images
+       as well as a webserver hosting the <literal>Helm Chart Repository</literal>
+       files.
+      </para>
+      <itemizedlist>
+       <listitem>
+        <para>
+         <literal>2</literal> Host machines for the mirror servers.
+        </para>
+        <itemizedlist>
+         <listitem>
+          <para>
+           SLES 15
+          </para>
+         </listitem>
+         <listitem>
+          <para>
+           2 (v)CPU
+          </para>
+         </listitem>
+         <listitem>
+          <para>
+           8 GB RAM
+          </para>
+         </listitem>
+         <listitem>
+          <para>
+           500 GB Storage
+          </para>
+         </listitem>
+        </itemizedlist>
+       </listitem>
+      </itemizedlist>
+     </listitem>
+    </varlistentry>
+   </variablelist>
+   <important>
+    <title>Adjust Number Of Mirror Servers</title>
+    <para>
+     This scenario description does not contain any fallback contingencies for
+     the mirror servers. Add additional mirror servers behind a loadbalancer if
+     you require additional reliability/availability.
+    </para>
+   </important>
+  </sect2>
+
+  <sect2 xml:id="sec.deploy.scenarios.airgap.network">
+   <title>Networking</title>
+   <para>
+    All members of the &productname; cluster must be able to communicate with
+    the internal mirror server(s) within the airgapped network.
+   </para>
+   <para>
+    Ports:
+   </para>
+   <itemizedlist>
+    <listitem>
+     <para>
+      80 HTTP - RMT Server and Helm Chart Repository Mirror
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      443 HTTPS - RMT Server and Helm Chart Repository Mirror
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      5000 HTTP - Docker Registry
+     </para>
+    </listitem>
+   </itemizedlist>
+   <note>
+    <title>Additional Port Configuration</title>
+    <para>
+     If you choose to add more Docker registries to your internal network,
+     these must run on different ports than the standard
+     <literal>5000</literal>.
```

Comment:
Another solution is to put something like HAProxy or Apache acting as a reverse proxy that handles vhosts. Should we dig into that? I would keep things easy for the 1st iteration, but it's your call

Reply:
I assume that people will need more than the suse registry in the long run. So if this makes using it easier, I'm all for it. That's something engineering has to decide with Product management if we want to support that in the long run.

Reply:
Yes, I agree people will need more than the SUSE registry. We can definitely provide something to address this need.

Let's keep things simple right now and not dig into that.